js/src/components/app-text/index.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import './index.scss';
+
+/**
+ * Renders a paragraph with some predefined styles based on the variant
+ *
+ *
+ * ```jsx
+ * <AppText variant="title-small">
+ * 		My text
+ * </AppText>
+ * ```
+ *
+ * @param {Object} props Component props
+ * @param {string} [props.variant] The variant to use for the text
```

Comment:
:nail_care: As the list and names of variants are defined here and closed. I think it's worth documenting available options.

Reply:
Thanks. Updated in https://github.com/woocommerce/google-listings-and-ads/pull/1779/commits/2207a72250f8e22d7aade03e54f4d20bce4c1d56